apps/aecore/lib/aecore/naming/name.ex

```diff
@@ -101,9 +101,11 @@ defmodule Aecore.Naming.Name do
 
   @spec decode_from_list(integer(), list()) :: {:ok, Name.t()} | {:error, reason()}
   def decode_from_list(@version, [owner, expires, status, client_ttl, pointers]) do
+    {:ok, decoded_owner} = Identifier.decode_from_binary(owner)
```

Comment:
You should:
- check for errors here
- check the type of identifier you decode. There is a a nice function to do that: `Identifier.decode_from_binary_to_value`